src/Moryx.ControlSystem.VisualInstructions.Endpoints/Models/Converter.cs

```diff
@@ -8,7 +8,7 @@ namespace Moryx.ControlSystem.VisualInstructions.Endpoints
 {
     internal static class Converter
     {
-        internal static ICustomSerialization _serialization = new PossibleValuesSerialization(null, new EmptyValueProvider());
+        internal static ICustomSerialization _serialization = new PossibleValuesSerialization(null, null, new EmptyValueProvider());
```

Comment:
Please check in product how the container/serviceprovider where added here, I'd prefer that instead of just adding null